bindings/matrix-sdk-ffi/src/timeline/content.rs

```diff
@@ -381,7 +394,49 @@ impl From<&matrix_sdk_ui::timeline::AnyOtherFullStateEventContent> for OtherStat
                 };
                 Self::RoomName { name }
             }
-            Content::RoomPinnedEvents(_) => Self::RoomPinnedEvents,
+            Content::RoomPinnedEvents(c) => {
```

Comment:
This is a massive chunk, which I'd prefer to have in a separate function, and unit tested, if possible.

Reply:
Yeah, that's probably a good idea. Thanks!

Reply:
https://github.com/matrix-org/matrix-rust-sdk/pull/3801/commits/665c2c2f15dce87dd410bd250ca0a5bc81188503